modules/vstudio/tests/vc2010/test_globals.lua

```diff
@@ -253,3 +253,48 @@
 </PropertyGroup>
 		]]
 	end
+
+--
+-- Check that the "latest" systemversion works.
+-- note: we override the os.getWindowsRegistry method for reliable tests.
+--
+
+	function suite.windowsTargetPlatformVersionLatest_on2017()
+		p.action.set("vs2017")
+		systemversion "latest"
+		prepare()
```

Comment:
`prepare` is what will call `os.getWindowsRegistry`, so you'll need to put this just above the `test.capture`